CHANGELOG.md

```diff
@@ -4,6 +4,23 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.0.0 (2020-12-17)
+
+This is a major release concluding several release candidates and it delivers
+several major features to GA:
+* Add support for distributed hypertables
+* Add support for user-defined actions
+* Move enterprise features to community
+* Update Timescale License
+* Improve continuous aggregate API
+* Improve information views
+
+Since the last release candidate 4, it includes several minor improvements:
+* Properly block more unsupported functionalities on multinode
+* Improve and fix bugs in communication to data nodes
+* Optimize locking for create chunk API
+* Fix compression status in chunks view for distributed chunks
+
```

Comment:
Shouldn't we include "minor features", "bugfixes", etc. like we've done for previous release? Then also reference the PR numbers?

I think we should also add a "Thanks" section, if applicable.

Reply:
During my first iteration through all commits I haven't found any. I definitely missed them.